### DISCUSSION TITLE
Cause of death is unreadable

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#14958] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

cause of deaths is cutoff
this looks terrible

![image](https://github.com/user-attachments/assets/ee9222cd-ef74-45c7-b807-6bfdea249fd1)

### Reproduction steps

1. pvp

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
